src/pybind/mgr/dashboard/frontend/src/app/shared/enum/critical-confirmation-modal-impact.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum DeletionImpact {
+  normal = 'normal',
```

Comment:
Instead of `normal` we can go with `medium` following guidelines levels https://carbondesignsystem.com/community/patterns/remove-pattern/#delete-remove-variations
Also, constants in caps?
```suggestion
  MEDIUM = 'medium',
```

Reply:
okay, I can change this